dist/jquery.scrolling-tabs.js

```diff
@@ -1246,20 +1246,32 @@
                         $rightArrow);
     }
   
-    function getNewElTabAnchor(tab, propNames) {
-      return $('<a role="tab" data-toggle="tab"></a>')
-              .attr('href', '#' + tab[propNames.paneId])
-              .html(tab[propNames.title]);
+    function getNewElTabAnchor(tab, propNames, options) {
+      var $a = $('<a role="tab" data-toggle="tab"></a>')
+                .attr('href', '#' + tab[propNames.paneId])
+                .html(tab[propNames.title]);
+      if (options['cssClassTabsA']) {
+          $a.addClass(options['cssClassTabsA']);
```

Comment:
Instead of introducing new option `cssClassTabsA`, what do you think about just checking the existing BS4 flag, and if it's `true`, adding the `nav-link` class directly since we already know that's the class required by BS4?

Your approach makes it more flexible but requires the user to provide two more settings which I expect would always be the same for BS4, so it would simplify the API if we just hardcode those values.

What do you think?

Reply:
By the time opening the PR, I wasn't aware of the bootsrap4 option. So yes, I will.